torch/_dynamo/variables/torch.py

```diff
@@ -245,6 +241,16 @@ def call_function(
                 self.value,
                 source=self.source,
             ).call_function(tx, args, kwargs)
+        if self.value is torch.overrides.get_default_nowrap_functions:
+            # [Note: __torch_function__] we return empty here because we restrict
+            # the set of functions that we trace __torch_function__ on to
+            # functions outside of the actual set. Implementing this properly will require implementing
+            # some variable types to track and compare tensor getset descriptors
+            from .builder import SourcelessBuilder
+
+            return SourcelessBuilder()(
+                tx, torch.overrides.get_default_nowrap_functions()
```

Comment:
Do we need a guard for this?  Or is it a constant?

Reply:
It's a constant